README.md

```diff
@@ -0,0 +1,56 @@
+Instrumental .NET Agent
+================
+
+Instrumental is a [application monitoring platform](https://instrumentalapp.com) built for developers who want a better understanding of their production software. Powerful tools, like the [Instrumental Query Language](https://instrumentalapp.com/docs/query-language), combined with an exploration-focused interface allow you to get real answers to complex questions, in real-time.
+
+This agent supports custom metric monitoring for .NET applications.
+
+Getting Started
+===============
+
+If you are using NuGet, add Instrumental.NET to your packages.config:
+
+```xml
+<id="Instrumental" version="0.2.0" targetFramework="net45" />
+```
+
+Or, download [Instrumental.dll](https://github.com/Instrumental/Instrumental.NET/releases/latest).
+
+Visit [instrumentalapp.com](https://instrumentalapp.com) and create an account, then initialize the agent with your API token, found in the Docs section.
+
+Simple Example
+==============
+
+Here are the basic Instrumental monitoring commands:
+
+```csharp
+using Instrumental;
+
+var agent = new Agent("project api token here");
+
+agent.Increment("myapp.logins");
+agent.Gauge("myapp.server1.free_ram", 1234567890);
+agent.Notice("Server maintenance window", TimeSpan.FromMinutes(15));
+Func<string> action = () => { DoLongRunningAction(); return "everything is fine"; };
+String actionResult = agent.Time("myapp.expensive_operation", action);
+```
+
+Worker Example
+==============
+
+You can easily use Instrumental Agent with background workers too, in this somewhat contrived timing example.  Note that Functions passed to Time must return a value or be defined as a Func<T> with a type:
```

Comment:
This might be a dumb question, but do we still need the comment about return values since adding the various `time` methods with various signatures?